.github/workflows/java-windows.yml

```diff
@@ -18,5 +18,5 @@ jobs:
                     java-version: ${{ matrix.java_version }}
             -   name: Build with Gradle
                 run: ./gr jvmBuild
-            -   name: build sample atrium+spek project
-                run: samples\jvm\spek\gradlew -p samples\jvm\spek build
+            -   name: build sample atrium+junit5 project
```

Comment:
Certainly a mistake. You have to add junit in addition and not remove spek. Also, check the indentation

Reply:
i deleted that branch and starting over, I'm not that good with git

Reply:
Maybe it's best to start over again then. Make sure you add .gitignore first to avoid that files in build, .gradle or .idea is added to a commit. (Consider to change your github password in case you stored passwords plaintext in your .idea folder, they are now public)